src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -298,6 +298,27 @@ class WorkItemListPage {
 
 
   }  
+  clickUserStoryWItype  ()  {
```

Comment:
Hi @ldimaggi ,
I used this approach to save lengthy codes and re-usability  of one generic method. But if above approach looks good . I have modified the methods with your suggested example .
Thanks :)